3-box-model/styles.css

```diff
@@ -0,0 +1,32 @@
+*{
+    font-family: 'Times New Roman', Times, serif;
+    font-size: large;
+}
+footer {
+    background-color: black;
+    border: 2px solid orange;
+    bottom: 0px;
+    color: whitesmoke;
+    position: fixed;
+    width: 100%;
+}
+header {
+    background-color: black;
+    border: 2px solid green;
+    color: whitesmoke;
+    position: sticky;
+}
+nav {
+    border: 2px solid blue;
+    position: absolute;
+    box-sizing: border-box;
+    padding-right: 90.5px;
+    display: grid;
+}
+section {
+    border: 2px solid red;
+    margin-left: 200px;
+}
+section:nth-last-child(3) {
```

Comment:
nice to see you understand last child but this is not making the content of the section visible, you probably need to set a padding for all the content sections.